libraries/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -690,4 +702,23 @@ export class ChangeAction extends BaseRushAction {
   private _logNoChangeFileRequired(): void {
     console.log('No changes were detected to relevant packages on this branch. Nothing to do.');
   }
+
+  private _stageGitChanges(patterns: string[]): void {
+    try {
+      patterns.forEach((pattern) => {
+        child_process.execSync(`git add "${pattern}"`);
+      });
+    } catch (error) {
+      console.error(`Cannot stage git changes ${(error as Error).message}`);
+    }
+  }
+
+  private _stageAndCommitGitChanges(patterns: string[], message: string) {
+    try {
+      this._stageGitChanges(patterns);
```

Comment:
Why don't we inline this method call here too? Or otherwise call two separate functions in on the call site, because otherwise this looks a bit strange with one function extracted and one function inlined. So the code has different levels of abstraction.

Reply:
Inlined both of them using the `Utilities.executeCommand`